Header header
string command
float64 probability
